packages/react-magma-dom/src/components/Table/Table.stories.tsx

```diff
@@ -300,7 +300,7 @@ export const ControlledPagination = args => {
   );
 
   return (
-    <Card isInverse={args.isInverse}>
```

Comment:
I removed this because it seemed like cruft that us unintentionally redundant.

Reply:
We have this for testing purposes, so let's just keep it as is.